settings.gradle.kts

```diff
@@ -1,9 +1,27 @@
 pluginManagement {
     repositories {
-        mavenLocal()
-        gradlePluginPortal()
-        maven { url = uri("https://artifacts.itemis.cloud/repository/maven-mps/") }
-        mavenCentral()
+        val modelixRegex = "org\\.modelix.*"
```

Comment:
`excludeGroup` and `includeGroup` could be used instead of the variant with the Regex.

Reply:
It doesn't work everywhere like that, since `includeGroup` would not include subgroups like `org.modelix.mps`. I used the variant with the regex for consistency.

Reply:
Ah, ok. Then you could use `includeGroupAndSubgroups` (But Regex is also fine.)

Reply:
There's also `includeGroupAndSubgroups` but that one is marked as incubating so I avoided it.